### PR TITLE
Add .NET 5 sdk extension

### PR DESCRIPTION
**Pull request by tofudd**
### Please confirm your submission meets all the criteria
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x]  All assets referenced in the manifest are redistributable by any party. If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

This extension is fork of [org.freedesktop.Sdk.Extension.dotnet](https://github.com/flathub/org.freedesktop.Sdk.Extension.dotnet) but provides .NET 5 support. Original stays on version 3 because of compatibility reasons.

### CODE REVIEW

File: org.freedesktop.Sdk.Extension.dotnet5.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.freedesktop.Sdk.Extension.dotnet</id>
```
Comment (barthalion) [inline org.freedesktop.Sdk.Extension.dotnet5.appdata.xml:3]:
```suggestion
  <id>org.freedesktop.Sdk.Extension.dotnet5</id>
```

File: org.freedesktop.Sdk.Extension.dotnet5.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <name>.NET Core SDK extension</name>
```
Comment (barthalion) [inline org.freedesktop.Sdk.Extension.dotnet5.appdata.xml:6]:
```suggestion
  <name>.NET Core 5 SDK extension</name>
```
and so on in this file

**Comment by barthalion** [comment]
bot, build org.freedesktop.Sdk.Extension.dotnet5

**Comment by tofudd** [comment]
bot, build org.freedesktop.Sdk.Extension.dotnet5

**Comment by tofudd** [comment]
bot, build org.freedesktop.Sdk.Extension.dotnet5

**Comment by tofudd** [comment]
Sorry for this mess.

**Comment by tofudd** [comment]
bot, build org.freedesktop.Sdk.Extension.dotnet5

**Comment by nedrichards** [comment]
Is this a joint project with the people who are looking after version 3? What's the relation (if any)

**Comment by tofudd** [comment]
Previously I made pull request to their repo ([https://github.com/flathub/org.freedesktop.Sdk.Extension.dotnet/pull/13](https://github.com/flathub/org.freedesktop.Sdk.Extension.dotnet/pull/13)) but it was rejected. @refi64 suggested creating separate package with my changes.

**Comment by barthalion** [comment]
/merge

Out of pure curiosity, is it going to be used to package some app or just for development?

**Comment by tofudd** [comment]
I'm considering distributing one of my dotnet projects via flatpak.

**Pull request closed without merging**